CommandLine/libEGM/gmk.cpp

```diff
@@ -1187,7 +1219,13 @@ void LoadTree(Decoder &dec, TypeMap &typeMap, TreeNode* root) {
     const TypeCase type = RESOURCE_KIND[kind];
 
     // Handle postponed id->name references
-    dec.processPostoned(name, id, type);
+    dec.processPostponedName(name, id, type);
```

Comment:
Does name resolution actually have to happen, here? Protos just store name strings, right?